.env.example

```diff
@@ -6,6 +6,7 @@ DB_HOST=127.0.0.1
 DB_DATABASE=hackathon
 DB_USERNAME=homestead
 DB_PASSWORD=secret
+DB_CONNECTION=<mysql, pgsql, etc>
```

Comment:
By default, won't this break the app? Can you just make it one value like so @ProfNandaa :

`DB_CONNECTION=mysql` since we have pgsql as default.